test/runtests.jl

```diff
@@ -10,6 +10,18 @@ else
     typemax(Csize_t)
 end
 
+if haskey(ENV, "JULIA_TEST_EXEFLAGS")
+    const test_exeflags = `$(split(ENV["JULIA_TEST_EXEFLAGS"]))`
+else
+    const test_exeflags = `--check-bounds=yes --startup-file=no --depwarn=error`
+end
+
+if haskey(ENV, "JULIA_TEST_EXENAME")
+    const test_exename = `$(split(ENV["JULIA_TEST_EXENAME"]))`
```

Comment:
Sure... seems that I can use a private function for that. I doubt it's common though but why not....